src/views/confirmationPage.njk

```diff
@@ -1,10 +1,108 @@
 {% extends 'common/base.njk' %}
 {% set title = 'Confirmation Page' %}
 
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+
+{% set organisationTypeHtml %}
+<script>
```

Comment:
In this type of situation, the best practice to follow is to add a div with a specific id, group all the scripts together at the end of the file, and set the html content of this specific id with the associated value.
It's an example for organisation type:
- html to put: `<div id="organisationTypeContainer"></div>`
- script to run (at the end of the file): `document.getElementById("organisationTypeContainer").innerHtml = localStorage.getItem("organisationType");`

Like that, first, you will have only one script tag instead of multiple. Secondly, you put only html in your html block.

Reply:
This comment applies also for other fields.

Reply:
@di-fabs thanks for your feedback, I have applied the changes and updated the screenshot with the rendered result